### PR TITLE
Updated message types for github api

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**

### CODE REVIEW

**Comment by Dillon-Dumesnil** [comment]
@inventhouse

**Comment by Dillon-Dumesnil** [comment]
Relates to https://github.com/edx/edx-gomatic/pull/442 and https://openedx.atlassian.net/browse/LEARNER-6168

**Review by macdiesel — APPROVED** [review]
*shrug* sure!

File: tubular/scripts/message_prs_in_range.py [hunk 1]
```diff
@@ -66,33 +66,6 @@
...
-    help=u'(Deprecated, use "--release broke_vagrant")'
-)
-@click.option(
-    u'--release_e2e_failed', u'message_type',
-    flag_value=MessageType.e2e_failed.name,
-    help=u'(Deprecated, use "--release e2e_failed")'
-)
 @click.option(
     u'--release', u'message_type', type=click.Choice(
         [mt.name for mt in MessageType]
```
Comment (inventhouse) [inline tubular/scripts/message_prs_in_range.py]: Perhaps we should do a drive-by `sorted([mt.name for mt in MessageType])` here; it seems the enum doesn't keep them in declaration order so they're all jumbled in the `--help` message
